bindings/2.204/GeometryDash.bro

```diff
@@ -3520,7 +3523,7 @@ class FLAlertLayer : cocos2d::CCLayerColor {
 
 [[link(android)]]
 class FLAlertLayerProtocol {
-	virtual TodoReturn FLAlert_Clicked(FLAlertLayer*, bool);
+	virtual TodoReturn FLAlert_Clicked(FLAlertLayer*, bool) = win 0x1d240;
```

Comment:
looks sus

Reply:
check android but im pretty sure it just immediately returns. in that case its just some merged stub function so youre better off just inlining it:
`whatever flalertlayer_clicked(whatever, p2) {}`